client/js/socket-events/more.ts

```diff
@@ -17,13 +16,12 @@ socket.on("more", function (data) {
 			.filter((m) => m.self && m.text && m.type === "message")
 			.map((m) => m.text)
 			.reverse()
-			.slice(null, 100 - channel.inputHistory.length)
+			.slice(undefined, 100 - channel.inputHistory.length)
```

Comment:
Shouldn't this just be 0?

Reply:
`If start is undefined, then the slice begins at index 0.`

undefined is closer to null, which is what was there before. I don't personally care.